lib/matrix.rb

```diff
@@ -512,9 +523,9 @@ def freeze
   # * :strict_upper: yields only elements above the diagonal
   # * :upper: yields only elements on or above the diagonal
   #
-  #   Matrix[ [1,2], [3,4] ].each { |e| puts e }
-  #     # => prints the numbers 1 to 4
-  #   Matrix[ [1,2], [3,4] ].each(:strict_lower).to_a # => [3]
+  #     Matrix[ [1,2], [3,4] ].each { |e| puts e }
+  #       # => prints the numbers 1 to 4
+  #     Matrix[ [1,2], [3,4] ].each(:strict_lower).to_a # => [3]
```

Comment:
Why change the indent here?

Reply:
Otherwise, RDoc thinks the first line is the continuation of the list (same indent): https://docs.ruby-lang.org/en/master/Matrix.html#method-i-each
![image](https://user-images.githubusercontent.com/129656/67621831-43191200-f81c-11e9-9eff-dc0f8da6639b.png)